modules/ob-release-notes-1-6.adoc

```diff
@@ -0,0 +1,26 @@
+// This module is included in the following assembly:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-6_{context}"]
+= Release notes for {builds-shortname} 1.6
+
+{builds-shortname} 1.6 is now available on {ocp-product-title} 4.16, 4.17, 4.18, 4.19, 4.20.
```

Comment:
and 4.20. Reference link: https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.16/html/release_notes/gitops-release-notes#gitops-release-notes-1-16-0_gitops-release-notes